### ISSUE TITLE
pre-trained model

### ISSUE THREAD

**Issue by 582751489** [shepnerd/inpainting_gmcnn#59]
could you please offer more pretrained model on pytorch version.like random mask-ed face pictures？thanks so much

**Comment by shepnerd** [comment]
For pytorch version, we did not train more models on different datasets, except celeba-hq.
